src/main/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheck.java

```diff
@@ -196,49 +351,160 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   public void foo() {
+ *     int x = 1;
+ *     switch (x) { // OK
+ *       case 1:
+ *         // Some code
+ *         break;
+ *       default:
+ *         // Some code
+ *         break;
+ *     }
+ *
+ *     switch (x) { // Violation
+ *       case 1:
+ *         // Some code
+ *         break;
+ *       case 2:
+ *         // Some code
+ *         break;
+ *       default:
+ *         // Some code
+ *         break;
+ *     }
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check to produce a violation on a method with too many local variables:
  * </p>
  * <pre>
  * &lt;module name=&quot;DescendantToken&quot;&gt;
  *   &lt;property name=&quot;tokens&quot; value=&quot;METHOD_DEF&quot;/&gt;
  *   &lt;property name=&quot;limitedTokens&quot; value=&quot;VARIABLE_DEF&quot;/&gt;
  *   &lt;property name=&quot;maximumDepth&quot; value=&quot;2&quot;/&gt;
- *   &lt;property name=&quot;maximumNumber&quot; value=&quot;10&quot;/&gt;
+ *   &lt;property name=&quot;maximumNumber&quot; value=&quot;1&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   public void foo() { // OK
+ *     int var1 = 1;
+ *   }
+ *   public void boo() { // Violation
+ *     int var1 = 1;
+ *     int var2 = 2;
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check to produce a violation on a method with too many returns:
  * </p>
  * <pre>
  * &lt;module name=&quot;DescendantToken&quot;&gt;
  *   &lt;property name=&quot;tokens&quot; value=&quot;METHOD_DEF&quot;/&gt;
  *   &lt;property name=&quot;limitedTokens&quot; value=&quot;LITERAL_RETURN&quot;/&gt;
- *   &lt;property name=&quot;maximumNumber&quot; value=&quot;3&quot;/&gt;
+ *   &lt;property name=&quot;maximumNumber&quot; value=&quot;2&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   public int foo(int x) { // OK
+ *     if (x == -1) {
+ *         return -1;
+ *     } else if (x == 0) {
+ *           return 0;
+ *     }
+ *   }
+ *   public int boo(int x) { // Violation
+ *     if (x == -1) {
+ *         return -1;
+ *     } else if (x == 0) {
+ *         return 0;
+ *     } else {
+ *         return x;
+ *     }
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check to produce a violation on an interface with too many fields:
  * </p>
  * <pre>
  * &lt;module name=&quot;DescendantToken&quot;&gt;
  *   &lt;property name=&quot;tokens&quot; value=&quot;INTERFACE_DEF&quot;/&gt;
  *   &lt;property name=&quot;limitedTokens&quot; value=&quot;VARIABLE_DEF&quot;/&gt;
  *   &lt;property name=&quot;maximumDepth&quot; value=&quot;2&quot;/&gt;
- *   &lt;property name=&quot;maximumNumber&quot; value=&quot;0&quot;/&gt;
+ *   &lt;property name=&quot;maximumNumber&quot; value=&quot;1&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public interface foo { // OK
```

Comment:
`public interface Foo { // OK`

apply same capitalization of class/interface names for other cases below, thre are same problem in last example.